From 4704b5319f7b2f25cfc3789a354eeae0c6486f4d Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Sun, 3 Sep 2017 11:49:47 +0300
Subject: [PATCH 1982/2241] fix: sdhci: xenon: Add errata ID for ap806 speed
 reduction

Add errata ID for ap806 low speed settings in Xenon driver.

Change-Id: If3660c6df37df90edb877966d025a6e674e381b1
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/43644
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 drivers/mmc/host/sdhci-xenon.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/mmc/host/sdhci-xenon.c b/drivers/mmc/host/sdhci-xenon.c
index 9f399c4..d5bce31 100644
--- a/drivers/mmc/host/sdhci-xenon.c
+++ b/drivers/mmc/host/sdhci-xenon.c
@@ -377,7 +377,9 @@ static int xenon_probe_dt(struct platform_device *pdev)
 	u32 sdhc_id, nr_sdhc;
 	u32 tuning_count;
 
-	/* Disable HS200 on Armada AP806 */
+	/* Disable HS200 on Armada AP806
+	 * Errata 3.5: HWE-7296210
+	 */
 	if (of_device_is_compatible(np, "marvell,armada-ap806-sdhci"))
 		host->quirks2 |= SDHCI_QUIRK2_BROKEN_HS200;
 
-- 
2.7.4

